coalib/collecting/Collectors.py

```diff
@@ -110,6 +110,9 @@ def icollect_bears(bear_dirs, bear_globs, kinds, log_printer):
     """
     for bear_dir, dir_glob in filter(lambda x: os.path.isdir(x[0]),
                                      icollect(bear_dirs)):
+        # Since we get a real directory here and since we
+        # pass this later to iglob, we need to escape this.
+        bear_dir = glob_escape(bear_dir)
```

Comment:
yes

Sincerely,

Lasse Schuirmann

contact@viperdev.io
http://viperdev.io/

2016-03-20 14:27 GMT+01:00 Adhityaa Chandrasekar <notifications@github.com>:

> In coalib/collecting/Collectors.py
> <https://github.com/coala-analyzer/coala/pull/1644#discussion_r56764428>:
>
> > @@ -110,6 +110,9 @@ def icollect_bears(bear_dirs, bear_globs, kinds, log_printer):
> >      """
> >      for bear_dir, dir_glob in filter(lambda x: os.path.isdir(x[0]),
> >                                       icollect(bear_dirs)):
> > +        # Since we get a real directory here and since we
> > +        # pass this later to iglob, we need to escape this.
> > +        bear_dir = glob_escape(bear_dir)
>
> So basically break this commit into three, right?
>
>    - First with this bug fix
>    - The coala_delete_orig.py one
>    - Then the others (as they are all file collection using glob_list
>    instead of path_list)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1644/files/56293f729f1180450212e406294347643a0af802..a8a89993f5b5ce20d6e46cc9f511555620eee61b#r56764428>
>